src/librustdoc/clean/inline.rs

```diff
@@ -189,14 +189,12 @@ crate fn build_external_trait(cx: &DocContext<'_>, did: DefId) -> clean::Trait {
     let generics = (cx.tcx.generics_of(did), predicates).clean(cx);
     let generics = filter_non_trait_generics(did, generics);
     let (generics, supertrait_bounds) = separate_supertrait_bounds(generics);
-    let is_spotlight = load_attrs(cx, did).clean(cx).has_doc_flag(sym::spotlight);
```

Comment:
We should probably rename this flag to `notable_trait` since (a) it's clearer and (b) that's what the popup that it triggers is called. We should probably do it in another PR though; I'll try to do that soon.

Reply:
Opened #80936.